mycore-mets/src/main/java/org/mycore/mets/iiif/MCRMetsMods2IIIFConverter.java

```diff
@@ -159,7 +162,8 @@ public MCRIIIFManifest convert() {
 
             String identifier = this.physicalIdentifierMap.get(physicalSubDiv);
             try {
-                MCRIIIFImageInformation information = imageImpl.getInformation(identifier);
+                MCRIIIFImageInformation information = imageImpl
+                    .getInformation(URLDecoder.decode(identifier, StandardCharsets.UTF_8.toString()));
```

Comment:
Do not use .toString() here, to prevent  UnsupportedEncodingException to be thrown.

Reply:
UTF_8 is garanteed by all Java implementations
(https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html)
But the code looks still better without .toString()

Reply:
> UTF_8 is garanteed by all Java implementations

Yes but you have to use the [right method](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/URLDecoder.html#decode(java.lang.String,java.nio.charset.Charset) ), to prevent the Exception.

Reply:
Thanks for the hint, fixed now.